torchgeo/trainers/base.py

```diff
@@ -35,6 +35,12 @@ def __init__(self, ignore: Sequence[str] | str | None = None) -> None:
             ignore: Arguments to skip when saving hyperparameters.
         """
         super().__init__()
+        if ignore is None:
+            ignore = ['ignore']
+        elif isinstance(ignore, str):
+            ignore = [ignore, 'ignore']
+        else:
+            ignore = [*list(ignore), 'ignore']
```

Comment:
I think this deserves its own PR, I don't fully understand why this is needed.

Reply:
This is described in the first few comments of the PR